server/models/database_init.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+# Prerequisites
+psql -V
+service postgresql status
+
+# Postgres commands
+psql -U postgres -c 'CREATE DATABASE citizenapp;' -c '\c citizenapp' -c '\i ./database.sql'
+
+echo END database_init.sh
```

Comment:
You should add a newline to the end of your file. You can make this a setting in your editor.

Reply:
Done. Thanks!